### PR TITLE
Update readme with r2.0

### PR DESCRIPTION
**Pull request by steventk-g**
In this change:

* Add references to r2.0 wheels
* Sort cuda wheels by version, descending

### CODE REVIEW

**Review by ManfeiBai — APPROVED** [review]
LGTM

**Review by yeounoh — APPROVED** [review]
LGTM

**Review by alanwaketan — APPROVED** [review]
LGTM.

File: README.md [hunk 1]
```diff
@@ -76,52 +76,66 @@ The following pre-built docker images are available. For running dockers, check
...
+2.0 | `gcr.io/tpu-pytorch/xla:r2.0_3.8_tpuvm` |
 1.13 | `gcr.io/tpu-pytorch/xla:r1.13_3.8_tpuvm` |
-1.12 | `gcr.io/tpu-pytorch/xla:r1.12_3.8_tpuvm` |
 nightly | `gcr.io/tpu-pytorch/xla:nightly_3.8_tpuvm` |
 nightly at date | `gcr.io/tpu-pytorch/xla:nightly_3.8_YYYYMMDD` |
 
 <br/>
 
-| Version | GPU CUDA 11.2 + Python 3.7 Docker |
+| Version | GPU CUDA 11.8 + Python 3.8 Docker |
```
Comment (miladm) [inline README.md:86]: do we want to also mentioned 11.7 as a supported version here? any thoughts?

Comment (steventk-g) [reply README.md:86]: There's a separate section starting on line 94 for CUDA 11.7, but I'm open to organizing these tables however you think would be clearest

File: README.md [hunk 2]
```diff
@@ -76,52 +76,66 @@ The following pre-built docker images are available. For running dockers, check
...
+| Version | GPU CUDA 11.2 + Python 3.8 Docker |
 | --- | ----------- |
-| nightly | `gcr.io/tpu-pytorch/xla:nightly_3.8_cuda_11.7` |
-| nightly at date(>=20230210) | `gcr.io/tpu-pytorch/xla:nightly_3.8_cuda_11.7_YYYYMMDD` |
+| 1.13 | `gcr.io/tpu-pytorch/xla:r1.13_3.8_cuda_11.2` |
 
 <br/>
 
-| Version | GPU CUDA 11.8 + Python 3.8 Docker |
+| Version | GPU CUDA 11.2 + Python 3.7 Docker |
```
Comment (miladm) [inline README.md:108]: why does this say 11.2?

**Review by miladm — COMMENTED** [review]
Thanks @steventk-g

File: README.md (hunk 2 above)
Comment (steventk-g) [reply README.md:108]: These are our old 11.2 images from PTXLA 1.13 and earlier

Comment (miladm) [reply README.md:108]: looks good. thanks for clarifying.

**Review by miladm — APPROVED** [review]

**Pull request merged**